xdsl/dialects/csl/csl.py

```diff
@@ -313,6 +313,16 @@ class PtrType(ParametrizedAttribute, TypeAttribute, ContainerType[Attribute]):
     kind: ParameterDef[PtrKindAttr]
     constness: ParameterDef[PtrConstAttr]
 
+    @staticmethod
+    def get(typ: Attribute, is_single: bool, is_const: bool):
```

Comment:
Maybe `typ` could be a `TypeAttribute`, since `PtrType.type` is?

Reply:
done as suggested

Reply:
Changing back to Attribute to fix tests